adapters/deepintent/usersync_test.go

```diff
@@ -28,7 +28,7 @@ func TestDeepintentSyncer(t *testing.T) {
 	})
 
 	assert.NoError(t, err)
-	assert.Equal(t, "https://cdn.deepintent.com/syncpixel.html?gdpr=A&us_privacy=C&url=https%3A%2F%2Flocalhost%3A8888%2Fsetuid%3Fbidder%3Ddeepintent%26gdpr%3DA%26gdpr_consent%3DB%26uid%3D%5Bio_cid%5D", syncInfo.URL)
+	assert.Equal(t, "https://match.deepintent.com/usersync/136?id=unk&gdpr=A&us_privacy=C&url=https%3A%2F%2Flocalhost%3A8888%2Fsetuid%3Fbidder%3Ddeepintent%26gdpr%3DA%26gdpr_consent%3DB%26uid%3D%5Bio_cid%5D", syncInfo.URL)
```

Comment:
I noticed you added an id. I'd like to verify that this is identifying prebid traffic in general and is not specific to a host, like AppNexus/Xandr.

Reply:
This is for identifying traffic in general.

Reply:
Thank for confirming.